public/app/features/dashboard/components/DashboardRow/DashboardRow.test.tsx

```diff
@@ -39,19 +42,21 @@ describe('DashboardRow', () => {
   });
 
   it('should have two actions as admin', () => {
-    expect(wrapper.find('.dashboard-row__actions .pointer')).toHaveLength(2);
+    expect(screen.getByTestId('dashboard-row-actions').children).toHaveLength(2);
```

Comment:
Yep this sounds good! The "buttons" don't actually have roles because they are anchor tags with `onClick`... so I will add roles to them and also `aria-label`s.

Reply:
obvious question is why are these anchors when they probably should be buttons 🤔 i totally get that it's not the point of this task though so we can always fix that later 👍

Reply:
💯 
I tried changing them to `IconButton`, but the styling would need tweaking, so decided to just revert it and we can fix that another time 😅